code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -689,6 +689,51 @@
 		icon_state = "vox_keetquills"
 		species_allowed = list("Vox")
 
+// Apollo-specific
+
+	//Wryn antennae
+
+	wry_antennae_default

```

Comment:
See note above, also in this case the relative pathing makes the whole thing much easier to read.